scss/_tables.scss

```diff
@@ -32,3 +32,7 @@ th {
   padding: .825 * $em $em;
 }
 
+thead th {
+  border-bottom-width: 2px;
```

Comment:
maybe we should introduce some simple check into Gruntfile, which will search all `.scss` files and complain if `px` is found